src/Functions/splitByWhitespace.cpp

```diff
@@ -97,7 +97,30 @@ using FunctionSplitByWhitespace = FunctionTokens<SplitByWhitespaceImpl>;
 
 REGISTER_FUNCTION(SplitByWhitespace)
 {
-    factory.registerFunction<FunctionSplitByWhitespace>();
+    FunctionDocumentation::Description description = R"(
+Splits a string which is separated by whitespace characters into an array of substrings.
```

Comment:
Please include the note about `splitby_max_substrings_includes_remaining_string` in the md docs here as well.

Reply:
The same applies to all other `splitBy*` functions documented above^^.